.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+  - repo: https://github.com/pre-commit/mirrors-clang-format
+    rev: "v20.1.4"
```

Comment:
This is the current latest version.
https://github.com/pre-commit/mirrors-clang-format/releases/tag/v20.1.4